libs/windy-sounding/types/client/d.ts.files/broadcast.d.ts

```
@@ -114,17 +111,26 @@ export interface BasicBcastTypes<T extends keyof Plugins> {
   alertChanged: [];
 
   /**
-   * Indicates, that router has parsed URL and launched plugin
-   * that was requested in URL
+   * Indicates, that router has parsed URL
+   */
+  routerParsed: [PluginIdent | ExternalPluginIdent | void, ParsedStartupValues | undefined];
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)